### PR TITLE
Feature/se 49

### LINKED ISSUE #49: Create front-end for the Article
image

### PR DESCRIPTION
**Pull request by bishop6232**
closes SE2-WS23/documentation#49

BPnQGGY

implemented the Article page

### CODE REVIEW

File: src/pages/Article/ArticlePage.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import { Box } from '@mui/system';
+import PropTypes from 'prop-types';
+import { useTheme } from '@mui/material/styles';
+import mockArticle from './mockData';
+
+/**
+ * Renders the Article page.
+ * @returns {JSX.Element} ArticlePage component
+ */
+
```
Comment (Chrissydrx) [inline src/pages/Article/ArticlePage.jsx]: Please remove this empty line

File: src/pages/Article/ArticlePage.jsx
Comment (Chrissydrx) [inline src/pages/Article/ArticlePage.jsx:1]: image
Your scrollbar overlaps with the text and the color blue is not be best choice for a blue background

Comment (bishop6232) [reply src/pages/Article/ArticlePage.jsx:1]: The scroll bar overlap is fixed now but for the colour I asked product owner and Julius last Friday and they said the blue background is okay.  
On the other hand, we have only blue colours in our theme data.

Comment (julix14) [reply src/pages/Article/ArticlePage.jsx:1]: I think the color comment from Chris was more regarding the blue link on the blue background

Comment (bishop6232) [reply src/pages/Article/ArticlePage.jsx:1]: ok, I have changed that.

**Review by Chrissydrx — DISMISSED** [review]

**Review by Chrissydrx — COMMENTED** [review]
I'm not a big fan of this design. I think this is counter intuitive to use and a waste of space that can be used better. Also, It would better, if your article displays markdown rather than string, because so you have more option and styling options. But I mean it is so in the Figma. Anyways, I think you also overcomplicated your design structure, but it works, so if someone wants to approve this, I'm fine with this. Maybe @NicolasKmbr should have look about this PR to determine if this meets his requirements.

**Comment by julix14** [comment]
@NicolasKmbr - have look about this PR to determine if this meets your requirements.

**Comment by Chrissydrx** [comment]
@NicolasKmbr

**Comment by NicolasKmbr** [comment]
The design is okay, but please change the border-styling, I don't think it fits into our overall design.

**Review by NicolasKmbr — APPROVED** [review]

**Pull request merged**